minimumSizeSubarraySum.py

```diff
@@ -0,0 +1,23 @@
+class Solution(object):
+    def minSubArrayLen(self, s, nums):
+        """
+        :type s: int
+        :type nums: List[int]
+        :rtype: int
+        """
+        if sum(nums) < s:
+            return 0
+        res = len(nums)
+        i, j = 0, 0
+        while j < len(nums) and sum(nums[i : j + 1]) < s:
+            j += 1
+        res = min(res, j - i + 1)
+        while j < len(nums):
+            i = j - res + 1
+            if sum(nums[i : j + 1]) >= s:
```

Comment:
Do you really need this `if` given that you are doing the same check in the following `while`?

Reply:
You have not addressed previous comments/

Reply:
If is needed, because I need to update the res.